tslint.json

```diff
@@ -1,32 +1,13 @@
 {
-  "extends": [
-    "tslint:recommended"
-  ],
+  "extends": ["tslint:recommended"],
   "rules": {
     "arrow-parens": false,
-    "interface-name": [
-      true,
-      "never-prefix"
-    ],
-    "max-classes-per-file": [
-      false
-    ],
-    "member-access": [
-      false,
-      "check-accessor",
-      "check-constructor"
-    ],
+    "interface-name": [true, "never-prefix"],
+    "max-classes-per-file": false,
+    "member-access": [false, "check-accessor", "check-constructor"],
     "object-literal-sort-keys": false,
-    "semicolon": [
-      true,
-      "never",
-      "ignore-interfaces",
-      "ignore-bound-class-methods"
-    ],
+    "semicolon": [true, "never", "ignore-interfaces", "ignore-bound-class-methods"],
```

Comment:
- [ ] Find out why this is giving me an error about the wrong number of arguments.